R/dictionaries.R

```diff
@@ -193,22 +193,6 @@ dictionary.default <- function(x, file = NULL, format = NULL,
                                               # for compatibility
 }
 
-#' @export
-dictionary.dictionary2 <- function(x, file = NULL, format = NULL,
```

Comment:
Well, the function is needed because all of our constructors should work on their own object types. Either for updating them to the newest versions, or just for complete consistency.